### PR TITLE
Fix crashes when opening Play store or services dependent settings

### PR DESCRIPTION
**Pull request by quh4gko8**
Missing signature check at the moment, will be followed up in next commits

### CODE REVIEW

**Comment by quh4gko8** [comment]
https://github.com/GrapheneOS/platform_packages_modules_HealthFitness/blob/e2c9ef4e9ac435f7924eab3dd7e3c633afd7165a/apk/src/com/android/healthconnect/controller/permissions/shared/HelpAndFeedbackFragment.kt#L103-L105 is the reference for hiding these preferences

**Comment by quh4gko8** [comment]
Requires https://github.com/GrapheneOS/platform_frameworks_base/pull/423

File: apk/src/com/android/healthconnect/controller/utils/DeviceInfoUtils.kt [hunk 1]
```diff
@@ -54,6 +56,9 @@ class DeviceInfoUtilsImpl @Inject constructor() : DeviceInfoUtils {
                 if (!TextUtils.isEmpty(info.activityInfo.packageName)) {
                     try {
                         val ai = pm.getApplicationInfo(info.activityInfo.packageName, 0)
+                        if (GmsCompat.isGmsApp(ai)) {
+                            return FEEDBACK_REPORTER
```
Comment (muhomorr) [inline apk/src/com/android/healthconnect/controller/utils/DeviceInfoUtils.kt]: Should check first whether package name matches. isGmsApp() returns true for several GMS components, not only GmsCore (Play services).

Comment (quh4gko8) [reply apk/src/com/android/healthconnect/controller/utils/DeviceInfoUtils.kt]: Fixed in the next force-push

**Pull request merged**